lib/src/link_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:matrix_link_text/link_text.dart';
+import 'package:url_launcher/link.dart';
+
+class LinkWidget extends StatefulWidget {
```

Comment:
Well, for web it was done this way before too. @krillefear and I thought the advantage of semantic links is higher than properly inlining the contents.

A way more complex solution fulfilling both requirements would be re-implementing the `Link` widget from `package:url_launcher/link.dart` as a `InlineSpan` element. The question is whether it's worth the work? It would be pretty complex.

Reply:
What about opening an issue upstream to also have a LinkSpan and then roll with Link for now? ^^

Reply:
aaaah just saw you already did